app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -147,7 +144,7 @@ public void onTimelineChanged(Timeline timeline, int reason) {
      * @param context The associated context
      * @return A configured factory for Exoplayer
      */
-    private DefaultRenderersFactory configureExoplayerFactory(Context context) {
+    private DefaultRenderersFactory createExoplayerRenderersFactory(Context context) {
```

Comment:
@DavidFair does this rename look okay? imo it makes sense when it's called with `exoPlayerBuilder.setRenderersFactory(createExoplayerRenderersFactory(activity));`

Reply:
Nevermind, I'm gonna change the name back and have it return the exoplayer builder.

Then any future factory stuff (like setting a track selector) can go inside that function